pkg/cmd/extension/command.go

```diff
@@ -219,6 +221,33 @@ func NewCmdExtension(f *cmdutil.Factory) *cobra.Command {
 				return nil
 			},
 		},
+		&cobra.Command{
+			Use:   "run <name> [args]",
+			Short: "Run an installed extension",
+			Long: heredoc.Doc(`
+			Run an extension using either the extension name, or the extension repository's
+			organization and name. For example, if the extension repository is
+			"owner/gh-extension", you can pass either "extension" or "owner/gh-extension".
+
+			Use the extension repository's organization and name to disambiguate extensions
+			that have the same name, or share a name with a built-in command.
+
+			All arguments after the extension name will be forwarded to the executable
+			of the extension.
+			`),
+			Example: heredoc.Doc(`
+			$ gh extension run extension
+			$ gh extension run owner/gh-extension
```

Comment:
I don't see yet a strong need to take a whole extension name as input, so could we strike that down and just take in short extension name as input?